extensions/amp-smartlinks/0.1/amp-smartlinks.js

```diff
@@ -88,6 +88,10 @@ export class AmpSmartlinks extends AMP.BaseElement {
    */
   runSmartlinks_() {
     this.getLinkmateOptions_().then((config) => {
+      if (!config) {
+        return;
+      }
```

Comment:
LGTM, curious though should the config be asserted that it exists?

Reply:
It's a good question, right now everything assumes a configuration with a specific shape will exist.

In production this isn't the case for some usages, who end up in the undefined path after retrieving configuration.

With this PR the page operates as expected and the functionality silently bails out.